projects/swimlane/ngx-cron/src/lib/ngx-cron.service.ts

```diff
@@ -122,7 +168,8 @@ export class NgxCronService {
     }
   }
 
-  getCronData(cron: string, period: Period): ICronData {
+  getCronData(cron: string, period: Period, lang, configOverrides): ICronData {
+    this.expressionDescriptorOptions.locale = lang || 'en';
     const e: any = new ExpressionDescriptor(cron, this.expressionDescriptorOptions);
```

Comment:
suggestion: instead of `any`, can we type this to `ExpressionDescriptor` or `typeof ExpressionDescriptor`?